metadata/org.testcontainers/testcontainers/1.19.8/reflect-config.json

```diff
@@ -1834,7 +1834,7 @@
             {
                 "name": "<init>",
                 "parameterTypes": [
-                    
+
```

Comment:
Looks like all the changes in this file are just empty new lines. If they are not necessary, please revert this change